coding-standards/elixir.md

```diff
@@ -99,3 +99,72 @@ Guidelines for applications using Phoenix:
 - Instead of [Router Path helpers](https://hexdocs.pm/phoenix/routing.html#path-helpers), prefer using the full path in the tests (e.g. `/api/rabbits`) to test that the route is correct.
 - Document APIs using [OpenAPI](https://github.com/open-api-spex/open_api_spex), cast and validate operations in controllers using the provided [plug](https://github.com/open-api-spex/open_api_spex#validating-and-casting-params), and test controllers using [OpenAPISpex.TestAssertions](https://github.com/open-api-spex/open_api_spex#validate-responses).
 - The [CI template](../templates/elixir-ci.yaml) includes a step to generate the Swagger UI and publish it to GitHub pages. Please refer to [this pr](https://github.com/open-api-spex/open_api_spex/pull/489) to configure the `ApiSpec` module so that it does not depend on a running `Endpoint` when generating the `openapi.json` file.
+
+## Step-by-step debugging with pry
+
+Elixir's interactive shell has `pry` built-in, which allows us to introduce breakpoints in our code. In order to use, add:
+
+```
+require IEx; IEx.pry
+```
+
+and then start Phoenix with the interactive shell:
+
+```
+iex -S mix phx.server
+```
+
+For example, if we wanted to add a breakpoint to the handler of this event:
+
+```
+  def handle(%DeregistrationProcessManager{}, %HostDeregistrationRequested{
+        host_id: host_id,
+        requested_at: requested_at
+      }) do
+    %DeregisterHost{host_id: host_id, deregistered_at: requested_at}
+  end
+```
+
+we would add the breakpoint as explained above:
```

Comment:
Maybe a bit more direct ? 
Add  `require IEx` and `IEx.pry()`  as a breakpoint.
Example:

What do you think?